projects/1.20/assets/modrinth-create-more-features/create_more_features/lang/zh_cn.json

```
@@ -0,0 +1,132 @@
+  "block.create_more_features.incomplete_signalling": "不完整的信号装置",
+  "block.create_more_features.lineman_table": "电工桌",
+  "block.create_more_features.master_contraptions_table": "装置师桌",
+  "block.create_more_features.netheralavs": "余烬熔岩块",
+  "block.create_more_features.pipe_table": "管材桌",
+  "block.create_more_features.railway_casing_slab": "列车机壳台阶",
+  "block.create_more_features.railway_casing_stairs": "列车机壳楼梯",
+  "block.create_more_features.railway_decorative_casing": "铁路装饰机壳",
+  "block.create_more_features.railway_decorative_casing_fence": "铁路装饰栅栏",
+  "block.create_more_features.railway_decorative_casing_fence_gate": "铁路装饰护板栅栏门",
+  "block.create_more_features.railway_decorative_casing_slab": "铁路装饰台阶",
+  "block.create_more_features.railway_decorative_casing_stairs": "铁路装饰外壳楼梯",
+  "block.create_more_features.railway_decorative_casing_trapdoor": "铁路装饰活板门",
+  "block.create_more_features.railway_decorative_casing_wall": "铁路装饰护墙板",
+  "block.create_more_features.refined_radiance_block": "光辉石块",
+  "block.create_more_features.refined_radiance_casing": "光辉石机壳",
+  "block.create_more_features.refined_radiance_casing_slab": "光辉石机壳台阶",
+  "block.create_more_features.refined_radiance_casing_stairs": "光辉石机壳楼梯",
+  "block.create_more_features.seller_of_mechanisms_table": "机械师桌",
+  "block.create_more_features.showcase": "展示柜",
+  "block.create_more_features.signalling": "信号装置",
+  "block.create_more_features.sulfur_basalt_ore": "硫磺玄武岩矿",
+  "block.create_more_features.sulfur_lamp": "硫磺灯",
+  "block.create_more_features.sulfur_lamp_up": "明亮的硫磺灯",
+  "block.create_more_features.sulfur_ore": "硫磺矿",
+  "block.create_more_features.sundries_table": "杂物桌",
+  "block.create_more_features.velcro": "velcro",
+  "block.create_more_features.workshops": "工坊",
+  "entity.minecraft.villager.create_more_features.buyer_of_mechanisms": "零件工",
+  "entity.minecraft.villager.create_more_features.confectioner": "糖果制造商",
+  "entity.minecraft.villager.create_more_features.contraptionstrader": "大机械师",
+  "entity.minecraft.villager.create_more_features.engineer": "工程师",
+  "entity.minecraft.villager.create_more_features.exchangerofmechanisms": "机械交换器",
+  "entity.minecraft.villager.create_more_features.gizmomaster": "机巧大师",
+  "entity.minecraft.villager.create_more_features.lineman": "电工",
+  "entity.minecraft.villager.create_more_features.pipefitter": "管道工",
+  "entity.minecraft.villager.create_more_features.sellerofmechanisms": "机械师",
+  "entity.minecraft.villager.create_more_features.sundriesseller": "杂货师",
+  "fluid.create_more_features.netheralavs": "余烬熔岩",
+  "gui.create_more_features.worked.label_work_in_progress": "正在制作中",
+  "item_group.create_more_features.create_more_features_tab": "机械动力：更多村民",
+  "item.create_more_features.coal_pin": "煤粉",
+  "item.create_more_features.dark_storage": "暗隐水盒",
+  "item.create_more_features.graphite": "石墨",
+  "item.create_more_features.gravitron": "引力仪",
+  "item.create_more_features.gravitron_base": "引力发生器基础",
```

```suggestion
  "item.create_more_features.gravitron_base": "引力仪基件",
```
gravitron与上方统一，这个方块在游戏内有什么用途